commit 98f581f6aa4f79dbad1d00271f334d813e0870c2
Author: Oren Givon <oren.givon@intel.com>
Date:   Mon Aug 28 10:33:38 2017 +0300

    iwlwifi: fix wrong struct for a000 device
    
    commit f7f5873bbd45a67d3097dfb55237ade2ad520184 upstream.
    
    The PCI ID (0x2720, 0x0070) was set with the config struct
    iwla000_2ax_cfg_hr instead of iwla000_2ac_cfg_hr_cdb.
    
    Fixes: 175b87c69253 ("iwlwifi: add the new a000_2ax series")
    Signed-off-by: Oren Givon <oren.givon@intel.com>
    Signed-off-by: Luca Coelho <luciano.coelho@intel.com>
    Cc: Thomas Backlund <tmb@mageia.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/wireless/intel/iwlwifi/pcie/drv.c b/drivers/net/wireless/intel/iwlwifi/pcie/drv.c
index 858765fed8f8..3416c6155996 100644
--- a/drivers/net/wireless/intel/iwlwifi/pcie/drv.c
+++ b/drivers/net/wireless/intel/iwlwifi/pcie/drv.c
@@ -576,7 +576,7 @@ static const struct pci_device_id iwl_hw_card_ids[] = {
 	{IWL_PCI_DEVICE(0x2720, 0x0000, iwla000_2ax_cfg_hr)},
 	{IWL_PCI_DEVICE(0x34F0, 0x0070, iwla000_2ax_cfg_hr)},
 	{IWL_PCI_DEVICE(0x2720, 0x0078, iwla000_2ax_cfg_hr)},
-	{IWL_PCI_DEVICE(0x2720, 0x0070, iwla000_2ax_cfg_hr)},
+	{IWL_PCI_DEVICE(0x2720, 0x0070, iwla000_2ac_cfg_hr_cdb)},
 	{IWL_PCI_DEVICE(0x2720, 0x1080, iwla000_2ax_cfg_hr)},
 #endif /* CONFIG_IWLMVM */
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1180_98f581f6aa4f79dbad1d00271f334d813e0870c2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1180_98f581f6aa4f79dbad1d00271f334d813e0870c2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
